server/routes/api/users.js

```diff
@@ -83,7 +83,7 @@ router.post("/login", (req, res) => {
                     username: user.username
                 };
 
-// Sign token
+                // Sign token
                 jwt.sign(
```

Comment:
Right now there is no way for the client to access a user's username or language. Perhaps you could add them in the response for the login route or add a separate route that takes a token and returns that user information.